src/core/Panel.js

```diff
@@ -13,14 +13,14 @@ class Panel {
     const configs = {
       welcome: {
         htmlFile: 'src/panels/welcome/welcome.html',
-        cssFile: 'src/styles/styles.css',
+        cssFile: 'src/styles/welcome.css',
         jsFile: 'src/panels/welcome/welcome.js',
         containerId: 'carbon-visualizer-welcome-panel',
         className: 'cv-panel--welcome'
       },
       results: {
         htmlFile: 'src/panels/results/results.html',
-        cssFile: 'src/panels/results/results.css',
+        cssFile: 'src/styles/results.css',
```

Comment:
Technically out of scope, but I found that every time I changed the panel, like toggling between welcome and results, it would add another redundant `style` tag with the same ID.

<img width="379" height="292" alt="Image" src="https://github.com/user-attachments/assets/3b1d138a-5aa9-4356-9afe-1062e156d7e7" />

We could add a check at the start of `loadPanelCSS()` to avoid injecting CSS that has already been loaded, the same way `loadCoreCSS` does.